lms/djangoapps/teams/models.py

```diff
@@ -75,10 +75,16 @@ def add_user(self, user):
             raise NotEnrolledInCourseForTeam
         if CourseTeamMembership.user_in_team_for_course(user, self.course_id):
             raise AlreadyOnTeamInCourse
-        return CourseTeamMembership.objects.create(
+        membership = CourseTeamMembership.objects.create(
             user=user,
             team=self
         )
+        return membership
```

Comment:
Ah, I'd been experimenting with some things and forgot to change that back. I'll fix it up.